internal/mesh/internal/controllers/sidecarproxy/builder/destinations_test.go

```diff
@@ -32,7 +32,8 @@ var (
 				},
 				Ports: map[string]*pbcatalog.WorkloadPort{
 					"tcp":  {Port: 7070, Protocol: pbcatalog.Protocol_PROTOCOL_TCP},
-					"tcp2": {Port: 8081, Protocol: pbcatalog.Protocol_PROTOCOL_TCP},
+					"tcp2": {Port: 7071, Protocol: pbcatalog.Protocol_PROTOCOL_TCP},
+					"grpc": {Port: 8081, Protocol: pbcatalog.Protocol_PROTOCOL_GRPC},
```

Comment:
a port named `grpc` was actually TCP so I added TCP to make it easier to check golden files and not get false positives when you see a port named GRPC and the protocol is not GRPC.